README.md

```diff
@@ -82,9 +97,29 @@ BONChain.new // [BONChain new] and [[BONChain alloc] init] also work
                        // and get back a dictionary of attributes
 
 ```
+</details>
 
 You can also create a local variable or property to save a partially-configured chain. All the chaining methods pass copies of the chain, so you don't have to worry about later changes clobbering earlier properties:
 
+```swift
+// Base Chain
+let birdChain = BONChain()
+    .lineHeightMultiple(1.2)
+    .font(UIFont.systemFontOfSize(17.0))
+    .string("bird")
+
+// Two chains with different colors
+// that inherit their parents' properties
```

Comment:
#nth nitpick like you wouldn't believe: the apostrophe here should be curly (`’`) like in the Obj-C version.